docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,253 @@
+## STEP 4: **UPDATE MANIFEST**
+
+### 1. Open your manifest
+Recommended to do so in **GlacierKit** (or in **VS code** with a ***"schema"***).
+
+### 2. Add your blobsfolder:
+Add the following line that will make **SMF** do a ***blobs operation*** and have it refer to your ***`blobs`*** folder
+   ```json
+   "blobsFolders": ["blobs"]
+   ```
+>Here’s how your full manifest file might look with a `blobsFolders` line added:
+   ```json
+   {
+	"$schema": "https://raw.githubusercontent.com/atampy25/simple-mod-framework/main/Mod%20Manager/src/lib/manifest-schema.json",
+	"version": "0.3.0",
+	"id": "Yourname.Yourmod",
+	"name": "YourCustomSuit",
+	"description": "Your description of what your mod does",
+	"authors": ["YourName"],
+	"frameworkVersion": "2.33.20",
+	"contentFolders": ["content"],
+	"blobsFolders": ["blobs"]
+}
+   ```
+:::info Schema
+Notice the `$schema` line at the top. This line helps **VS Code** recognize the structure and options available for the manifest, providing error-checking and code suggestions as you edit. **GlacierKit** includes these features by default, but installing **VS Code** is highly recommended for better flexibility and ease in editing complex files.
+:::
+ ### 3. Save & Deploy
+ Simply click the little save icon on top or click **ctrl+S** on your keyboard. Deploy your mod in **SMF** and your new image will appear in place of the default one.
+
+:::success
+That's it! You’ve successfully replaced the suit’s image in the game with your own custom picture. Now, when you load up the game, your new image will appear in place of the default one. Don't close **GlacierKit** yet if you already are moving on to the next article or want to try the test below. It'll make things alot quicker. 
+:::
+
+---
+
+With this final step complete, you’re all set! There are additional tips and options for managing **blobs** in your mod below if you’d like to explore further.
+
+---
+## More on Blobs
+These tips will help you manage multiple images and **blobsfolders**.
+
+
+### More Blobs images
+
+- If you want to add more images, remember to recreate the **file paths** by making more folders. 
+And place each image in its relevant folder
+
+examples:
+
+| **Assembly Path** (in-game)                    | Your **BlobsFolder** Structure                      |
+|--------------------------------------------|---------------------------------------------|
+| `images/unlockables_override/absolution_suit.jpg`             | `blobs/images/unlockables_override/absolution_suit.jpg`        |
+| `images/characters/male_suits/caruso_idk.png`            | `blobs/images/characters/male_suits/caruso_idk.png`       |
+| `icons/equipment/distraction_device_old.jpg`               | `blobs/icons/equipment/distraction_device_old.jpg`          |
+
+- For multiple images that have the same **file path** in their **assembly path**, simply place them within the same folders (e.g., images under `unlockables_override`). Don't forget to **rename them too**.
```

Comment:
This line basically doesn't make sense (file path? assembly path?) unless you have the definitions earlier, which are not correct. Also rename them to what?